### DISCUSSION TITLE
Радикальные меры

### DISCUSSION THREAD

**Discussion by egorovna-developments** [xtrime-ru/antizapret-vpn-docker#60] [Q&A]
Подскажите, как пустить через VPN вообще весь трафик, кроме доменов .ru, .рф и .su?

**Comment by xtrime-ru** [comment]
нужно сделать регулярку все домены кроме указанных и написать ее в include.regex.custom.txt

Что то типа такого: 

```regex
^.*(?<!\.ru|\.su|\.рф)$
```

https://regex101.com
<img width="291" alt="image" src="https://github.com/user-attachments/assets/6b154064-c8ad-4e71-9a2a-73e261d3c3ee">

**Reply by egorovna-developments** [reply]
Не работает. На speedtest.net не сменился ip.

**Reply by xtrime-ru** [reply]
Ща не могу протестить у себя. 

Тогда можно перечислить все доменные зоны в include.host.custom.txt кроме этих.
